.travis.yml

```diff
@@ -28,10 +30,8 @@ env:
   - CXX=g++-4.8 DB="maria mysql mysql2 postgres sqlite3 oracledb" KNEX_TEST_TIMEOUT=60000 ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe ORACLE_SID=XE OCI_LIB_DIR=/u01/app/oracle/product/11.2.0/xe/lib LD_LIBRARY_PATH=/u01/app/oracle/product/11.2.0/xe/lib
 
 addons:
-  postgresql: '9.4'
+  postgresql: '9.6'
```

Comment:
No thats not needed I just updated postgres used in tests... one can use knex with postgres 8 if one likes to. Just cannot use some newer features.